namron_dimmer.json

```diff
@@ -0,0 +1,338 @@
+{
+	"manufacturer": "Namron",
+	"manufacturerId": "0x0438",
+	"label": "Z-Wave dimmer 2 400W",
+	"description": "In-Wall Dimmer Module",
+	"devices": [
+		{
+			"productType": "0x0200",
+			"productId": "0xd00c"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"2": {
+			"label": "Save load state",
+			"description": "Saves load state before power failure",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 2,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+			"options": [
+			  {
+					 "label": "Shut off load",
+					 "value": 0
+			  },
+			  {
+					 "label": "Turn on load",
+					 "value": 1
+			  },
+			  {
+					 "label": "Save load state before power failure"
+					 "value": 2
+			  }
+		   ]	 
+		},
+		"3": {
+			"label": "Enable/disable to send report",
+			"description": " Sends the basic report to the Lifeline when the load state changed（When value set as 1, repower on the dimmer, it will send Basic report automatically）",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+			"options": [
+			  {
+					 "label": "Disable to send Basic report",
+					 "value": 0
+			  },
+			  {
+					 "label": "Enable to send Basic report",
+					 "value": 1
+			  }
+		   ]
+		},
+		"4": {
+			"label": "Default fade time",
+			"description": "This value has the same function as Duration of Multilevel. Valid value: 0~127",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 127,
+			"defaultValue": 1,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"5": {
+			"label": "Minimum brightness value",
+			"description": "Valid value: 0~50. The bigger the value is, the higher the load’s minimum brightness is",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 50,
+			"defaultValue": 15,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"6": {
+			"label": "Maxium brightness value",
+			"description": "Valid values: 0 ~ 100",
```

Comment:
Style guide:
> information like units or **available ranges should be removed** as that information is provided to UIs through other properties.

Reply:
Hi, and thanks for the input.
I have gone through the JSON code again.
After reading the Style Guide, it is my understanding that the parameters readOnly, writeOnly are not needed unless the manual states that the value is in fact Read Only?
And the allowManualEntry must be TRUE where the user can set a VALUE between given values, and must be set to FALSE where unchangable OPTIONS are set
And: What is the "unsigned": true for? I see it several places, but the Style Guide does not explain what it is.

I have also run the code through an online JSONLint checker, and it detects no errors now.
So, do you mind looking over the code once again?

Thank you